src/pages/admin/Event/EventStats/TableColumns.js

```diff
@@ -102,6 +102,23 @@ const getDefaultColumns = (eventID, eventYear, refreshTable, tableType) => [
       </div>
     )
   },
+  {
+    title: "Date Created",
+    field: "createdAt",
+    customSort: (a, b) => {
+      const timestampA = Date.parse(a.createdAt);
+      const timestampB = Date.parse(b.createdAt);
+      return timestampA - timestampB;
+    },
```

Comment:
Is this custom sort necessary? I think the `getTime()` from the DDB should have the createdAt stored in milliseconds, so it might already be sortable by default

Reply:
nit: what happens if we try to sort an older event that doesn't have `createdAt` in its registration entries? Will our frontend crash? This doesn't need investigation or a fix for now, just curious if it'd be doing `Date.parse(null)`

Reply:
For tables without any createdAt data there are no crashes.

Reply:
I think I was trying to keep `createdAt` sorting consistent with `updatedAt`. You're right the createdAt is by default sorted in descending order but if I remove the customSort function for createdAt it won't be able to sort in ascending order.